lib-es5/utils/encoding/sdkVersionID/encodeVersion.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var base64Map = require('./base64Map');
+var reverseVersion = require('./revereseVersion');
+
+/**
+ * @description Encodes a semVer-like version string
+ * @param {string} semVer Input can be either x.y.z or x.y
+ * @return {string}
+ */
+function encodeVersion(semVer) {
```

Comment:
@patrick-tolosa , what will happen if I'll send semVer=123456?

Reply:
There's a test file with a list of invalid tests and their expected results

    `expect(getSDKVersionID({}, 'abcdefg', 'abcdefg')).to.equal('E');
     expect(getSDKVersionID({}, '123456', '123456')).to.equal('E');
     expect(getSDKVersionID({}, '43.21.200', '43.21.26')).to.equal('E');`

When we don't know what to do with the input, we return the constant 'E'